eng/Directory.Packages.props

```diff
@@ -92,7 +92,7 @@
     <!--
       Subset of Microsoft.VisualStudio.Sdk meta-package (run `csi generate-vssdk-versions.csx` to update based on VSSDK meta-package).
     -->
-    <PackageVersion Include="Microsoft.ServiceHub.Framework" Version="4.5.31" />
+    <PackageVersion Include="Microsoft.ServiceHub.Framework" Version="4.5.35" />
```

Comment:
Shouldn't be updated without updating Microsoft.VisualStudio.SDK. Probably better to bump Microsoft.VisualStudio.SDK in a separate PR.

Reply:
How to determine the right VS SDK version to use. I saw the current version is `17.10.234-preview.1` but that one is not available on nuget?